cromulent/model.py

```diff
@@ -122,6 +122,7 @@ def __init__(self, base_url="", base_dir="", lang="", full_names=False,
 		self.linked_art_boundaries = False # break on linked art API boundaries between classes
 		self.id_type_label = True # references are id, type and _label, not just id.
 
+		self.json_serializer = "normal" # "normal" or "fast"
```

Comment:
Is there any documentation about when this can/should be set to `"fast"`? Are there situations were `"fast"` cannot be used?

Reply:
Before Python 3.6 it won't generate the right order, so system dependent if order is a requirement.

It also doesn't do the scoped context |s that I use for documentation purposes.

Other than that, I think it's a drop in replacement.

Reply:
OK. This shouldn't stop merging, but it would probably be good to add documentation or at least in-line comments about this.